ansible/roles/openondemand/defaults/main.yml

```
@@ -102,6 +102,9 @@ openondemand_osc_ood_defaults:
   ood_auth_openidc: "{{ openondemand_auth_defaults.oidc.ood_auth_openidc if (openondemand_auth | lower) == 'oidc' else none }}"
   httpd_auth: "{{ openondemand_auth_defaults[openondemand_auth | lower].httpd_auth }}"
 
+  # Use Ark pulp repos by default
```

```suggestion
  # Use repo file provided by dnf_repos by default
```

b/c it could be local Pulp, not Ark directly